quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -1471,4 +1471,17 @@ function getFileOverwritePermission() {
   fi
 }
 
+function isZitiEdgeControllerPortListening(){
```

Comment:
let's make this a more generic "isPortAvailable" that takes a parameter of the port to check. Probabaly need to turn that logic around then as well and need to pass the port to be checked and not hardcode the 1280.  Also need to add this to the `checkPrereqs` function. where it does something like check "if !isPortAvailable print error and exit"

Reply:
ok

Reply:
Please review now